### PR TITLE
Wizard War

### PR DESCRIPTION
**Pull request by MattZ45986**
This pull request features the mod Wizard War.  In light of the new Grumbledorf character in the latest update, I've made this minigame to show him off in a mod of his own. 

In the game, players may fight as a team or in a free-for-all, striving to be the last one standing.  It is similar to classic elimination, but there's a twist.  Instead of using bombs, these wizards produce orbs to blast opponents.  Each team/player creates a different colored sphere of energy, which they wield against their enemies.  The orbs can be made in an unlimited or limited amount, they may or may not explode on impact with other orbs, and running, punching,  jumping, and picking up can be enabled or disabled: it is all available in the settings menu.

In short, Bombsquad has not seen a new mod in quite some time, and I thought a new update was the perfect excuse to make one.

### CODE REVIEW

**Comment by Mrmaxmeier** [comment]
Nice work! Thanks :+1:
